src/components/ebay-listbox-button/template.marko

```diff
@@ -17,7 +17,8 @@
     <button
         w-id="button"
         class=[
-            "listbox__control",
+            "btn",
```

Comment:
The `.expand-btn` class didn't provide the correct styles? Do we just need to put these classes in a different order in the class attribute?

Reply:
Good catch! I had issues loading expanded button and I was trying something else. I'll revert it